core/cap-0042.md

```diff
@@ -52,7 +52,7 @@ Note that this CAP does not modify how transactions gets applied (ie: the apply
 
 ### XDR Changes
 
-```
+```diff
```

Comment:
The needs the attribute that specified the base ref. E.g.
```suggestion
```diff mddiffcheck.base=74498070b99a7fb1d18b78d104f95d797b4f4c2c
```
@MonsieurNicolas can you confirm the right base ref for this diff?

Reply:
I ended up opening https://github.com/stellar/stellar-protocol/pull/1099 : for whatever reason the diff was slightly off